.github/workflows/release.yaml

```diff
@@ -9,33 +11,50 @@ env:
 
 jobs:
   release-binaries:
-    name: Release ${{ matrix.arch }}-${{ matrix.os }}
-    runs-on: ubuntu-20.04
+    name: Release ${{ matrix.target }}
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        arch:
-          - aarch64
-          - x86_64
-        os:
-          - unknown-linux-gnu
-          - apple-darwin
+        include:
+          - target: x86_64-pc-windows-gnu
+          - target: x86_64-unknown-linux-gnu
+          - target: aarch64-unknown-linux-gnu
+          # see <https://github.com/cross-rs/cross-toolchains
+          # - target: x86_64-apple-darwin
+          # - target: aarch64-apple-darwin
     steps:
       - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+          target: ${{ matrix.target }}
       - uses: Swatinem/rust-cache@v1
         with:
           sharedKey: shared-cache
-      - name: Compile and release
```

Comment:
Thanks for kicking this one, been hurting me when I last looked at this project as well.